doc/nrf/ug_zigbee_configuring_zboss_traces.rst

```diff
@@ -0,0 +1,149 @@
+.. _ug_zigbee_configuring_zboss_traces:
+
+Configuring ZBOSS traces in |NCS|
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This ZBOSS stack comes in a set of precompiled libraries which can complicate the debugging process.
+To help with that, the ZBOSS stack can be configured to print trace logs that allow you to trace the stack behavior.
+This page describes how to enable and configure ZBOSS trace logs.
+
+Trace logs are printed in binary form and require access to stack source files, which are to be decoded into log messages.
+Enabling trace logs can help with the debugging process even if you cannot decode them (due to lack of access to stack source files) because of the information they can provide about the stack behavior whenever an issue is found or reproduced.
+
+
+.. note::
+     ZBOSS trace logs are meant to be used for debugging ZBOSS based applications.
+     These libraries are not certified and should not be used for production firmware or end products.
+
+
+
+Requirements
+************
+
```

Comment:
Do any hardware requirements need to be added here?
Or anything else?

Reply:
Not really.
The only thing I can think of is additional USB cable if trace logs via USB are enabled.